### PR TITLE
chore: use UI kit modal for Experiment actions

### PR DESCRIPTION
**Pull request by johnkim-det**
## Description

Move
Screen Shot 2023-04-05 at 3 00 07 PM

Bulk Move, No project selected
Screen Shot 2023-04-05 at 3 00 46 PM

Fork, Simple Config
Screen Shot 2023-04-05 at 3 01 07 PM

Fork, Full Config
Screen Shot 2023-04-05 at 3 01 19 PM

Continue Trial, Simple Config
Screen Shot 2023-04-05 at 3 01 31 PM

Continue Trial, Full Config
Screen Shot 2023-04-05 at 3 01 43 PM

Edit
Screen Shot 2023-04-05 at 3 02 13 PM

Delete
Screen Shot 2023-04-05 at 3 02 24 PM

Stop, Save Checkpoint
Screen Shot 2023-04-05 at 3 02 54 PM

Stop, Without Checkpoint Save
Screen Shot 2023-04-05 at 3 03 04 PM

## Test Plan

Regression for 
- [ ] Experiment List Row Action, Move
- [ ] Experiment List Bulk Action, Move
- Experiment Details Header
  - [ ] Move
  - [ ] Edit
  - [ ] Delete
  - [ ] Fork
- Experiment Details Header, Paused Experiment
  - [ ] Stop Experiment
  - [ ] Continue Trial 
- [ ] Trial Details Header, Continue Trial
## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

## Ticket

WEB-1101

### CODE REVIEW

File: webui/react/src/components/CreateGroupModal.tsx [hunk 1]
```diff
@@ -115,7 +115,7 @@ const CreateGroupModalComponent: React.FC<Props> = ({ onClose, users, group }: P
           groupRoles.map((r) => r.id),
         );
         if (!nameUpdated && !usersUpdated && !rolesUpdated) {
-          message.info('No changes to make.');
+          message.info('No changes to save.');
```
Comment (hkang1) [inline webui/react/src/components/CreateGroupModal.tsx:118]: good eyes (plural, not sure why people say "good eye" singular)

**Review by hkang1 — COMMENTED** [review]
The code looks great and cleaner!

Listing out things I've encountered:
- When opening an experiment create modal, the submit button is disabled even though the name field has a value. It's not until I click into the name field and then exit, does it enable the submit button. Screenshot 2023-04-06 at 10 33 20 AM
- There is a regression on this behavior - the form field of experiment fork name would get copied over to the advanced exp config field of `name` and vice versus (the config field of `name` gets copied back to the form when toggling back to Simple mode). `max_length` is in the same boat.
- There is a test file for Create modal, but not the rest of the modals. Would it be ok to add some rough basic tests for each of them?

File: webui/react/src/components/CreateGroupModal.tsx (hunk 1 above)
Comment (ClaireNeveu) [reply webui/react/src/components/CreateGroupModal.tsx:118]: The "eye" in "good eye" refers to the "attention or notice" definition of "eye", not the "ocular organ" definition. (This is related to the verb form)

Comment (hkang1) [reply webui/react/src/components/CreateGroupModal.tsx:118]: 🤩 didn't have an emoji for just one eye...

File: webui/react/src/components/ExperimentActionDropdown.tsx [hunk 2]
```diff
@@ -240,15 +232,29 @@ const ExperimentActionDropdown: React.FC<Props> = ({
         onOpenChange={onVisibleChange}>
         {children}
       </Dropdown>
-      {modalExperimentMoveContextHolder}
+      <ExperimentMoveModal.Component
+        {...{
```
[the remark this answers is not in the document]
Comment (johnkim-det) [reply webui/react/src/components/ExperimentActionDropdown.tsx:236]: was just avoiding extra typing when converting from function call. I'll go ahead and change it to pass props normally.

**Review by hkang1 — APPROVED** [review]
Thanks for adding the tests and the simple/advanced mode values are translated across each mode nicely!

**Review by ClaireNeveu — APPROVED** [review]
Design looks good!

**Pull request merged**